equipment_handler/source/tempsense_athena.c

```diff
@@ -23,25 +23,23 @@
 #define CHANNEL_LOCAL 0
 #define CHANNEL_REMOTE 1
 
-uint8_t tmp_addr[6] = {0x1D, 0x1E, 0x2A, 0x4C, 0x4F, 0x4E};
+uint8_t tmp_addr[6] = {0x49, 0x4A};
```

Comment:
we should really replace the value "6" everywhere with a constant `#define NUM_TEMP_SENSORS` and change it everywhere

Reply:
done